demos/apps/echo-with-auth/logic-app.yaml

```diff
@@ -0,0 +1,45 @@
+# Example service with an injected proxy
+apiVersion: v1
+kind: Service
+metadata:
+  name: logic
+  labels:
+    app: logic
+spec:
+  ports:
+  - port: 80
+    targetPort: 8080
+    name: http
+  - port: 8080
+    targetPort: 80
+    name: http-alternative
+  selector:
+    app: logic
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: logic
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: logic
+        version: unversioned
+    spec:
+      containers:
+      - name: app
+        image: docker.io/istio/app:2017-03-17-22.11.25
+        imagePullPolicy: Always
```

Comment:
I would suggest replacing this image with a simple curl based image from docker.io. That should be more than sufficient.